src/server/editorServices.ts

```diff
@@ -1,3 +1,9 @@
+namespace ts {

```

Comment:
Thats because UserPreferences is moved to compiler and when building server it wont have lazyConfiguredProjectsFromExternalProject field. We add that only when building server. (UserPreferences is in ts namespace as appose to ts.server)

Reply:
changed it a bit to use protocol.UserPreferences

Reply:
@andy-ms is more likely than I am to know what conventions we follow in this regard, but I find the new version clearer.  Thanks!